terraform/moe_user_read.tf

```diff
@@ -0,0 +1,39 @@
+resource "aws_iam_user" "create_moe_user_read" {
+  name = "moe_user_read"
+  path = "/system/"
```

Comment:
I'm not against putting this user in a path, but we (the devs) need to come to a consensus as to how we want to organize them.  I don't think `system` is a very specific or useful classification.

Reply:
I was just using the default example from terraform documentation. I have no problem changing it to something to be more uniform: https://www.terraform.io/docs/providers/aws/r/iam_user.html

Reply:
I wouldn't specify a path unless we have a scheme for organizing users.  (The path is optional and defaults to `/`.)